docs/contact.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Contact Us</title>
+</head>
+<body>
+    <div class="container">
+        <form>
+            <h1>Contact Us</h1>
+            <input type="text" value="Name:" placeholder="Name">
+            <input type="text" value="Email:" placeholder="Email">
+            <textarea name="Message:" id="" cols="30" rows="10"></textarea>
+            <button>Send Message</button>

```

Comment:
It's best to specify `type="submit"` for the button that is intended to be the submit button.